docs/manual.rst

```diff
@@ -923,7 +923,7 @@ libtorrent's point of view, it doesn't matter what it is. libtorrent only makes
 sure the peer certificates are signed by the correct root certificate.
 
 One way to create the certificates is to use the ``CA.sh`` script that comes
-with openssl, like thisi (don't forget to enter a common Name for the
+with openssl, like these (don't forget to enter a common Name for the
```

Comment:
I just landed a patch in RC_1_2 fixing these typos. this will conflict with that

Reply:
will revert this typo fix then